test/plugins/xterm.plugin.bats

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bats
+
+load ../test_helper
+load ../../lib/composure
+
+load ../../plugins/available/xterm.plugin
+
+function local_setup {
+  setup_test_fixture
+
+  # Copy the test fixture to the Bash-it folder
+  if command -v rsync &> /dev/null
```

Comment:
use `command_exists`, so you wont need to `/dev/null` it

Reply:
done